### PR TITLE
Implement Fire Window UI tests

### PR DESCRIPTION
**Pull request by jotaemepereira**
Task/Issue URL: https://app.asana.com/0/1204006570077678/1205717021705381/f
Tech Design URL:
CC:

**Description**:
Implement Fire Window UI tests 

**Steps to test this PR**:
Check that workflow passes on macOS 13 and 14: https://github.com/duckduckgo/macos-browser/actions/runs/11817794843

**Definition of Done**:

* [x] Does this PR satisfy our [Definition of Done](https://app.asana.com/0/1202500774821704/1207634633537039/f)?

---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -341,6 +341,18 @@ platform :mac do
     )
   end
 
+  desc 'Creates a new Kechain to use on UI tests'
+  lane :create_keychain_ui_tests do |options|
+    create_keychain(
+      name: "DefaultKeychain",
+      password: "default",
```
Comment (jotaemepereira) [inline fastlane/Fastfile:348]: Dominik, do you think it makes sense to create a random password even if we are not using it anywhere?

Comment (ayoy) [reply fastlane/Fastfile:348]: Only to make it look more professional perhaps 😅  fine to keep it as is 👍

File: UITests/FireWindowTests.swift [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+            let webViewFire = app.webViews.firstMatch
+            let webViewCoordinate = webViewFire.coordinate(withNormalizedOffset: CGVector(dx: 5, dy: 5))
+            webViewCoordinate.tap()
+            app.typeKey("\t", modifierFlags: [])
+            sleep(1)
+            let autoFillPopup = webViewFire.buttons["test@duck.com privacy-test-pages.site"]
+            let coordinate = autoFillPopup.coordinate(withNormalizedOffset: CGVector(dx: 0.5, dy: 0.5))
+            coordinate.tap()
+
+            /// On macOS there are some issues when accessing web view elements so we do not check the value of the email text field.
```
Comment (ayoy) [inline UITests/FireWindowTests.swift:137]: `On macOS 13`, right?

**Review by ayoy — APPROVED** [review]
LGTM, thanks @jotaemepereira!

File: UITests/FireWindowTests.swift (hunk 2 above)
Comment (jotaemepereira) [reply UITests/FireWindowTests.swift:137]: Yes 😅 . Fixed

**Pull request merged**